### DISCUSSION TITLE
Feature parity tracking

### DISCUSSION THREAD

**Discussion by phill-85** [patternfly-yew/patternfly-yew#40] [General]
Do you have a way of tracking the feature completeness / parity with Paternfly?

**Comment by ctron** [comment]
Unfortunately no. It also might be a bit problematic to get there, as it would be a moving target. PatternFly itself evolves all the time.

I absolutely understand that it would be nice to have, and if you're missing something in particular, please do open an issue. Some things can get added rather quickly, so being aware of such features helps prioritizing.

In any case, everything that PatternFly itself provides (not necessarily the ReactJS stuff) should be part of PatternFly Yew too. If it's missing, I would consider it a bug which needs fixing.
